modules/tinymce/changelog.txt

```diff
@@ -1,5 +1,5 @@
 Version 5.8.0 (TBD)
-
+    Added alpha list numbering support to the `lists` plugin #TINY-6891
```

Comment:
Should we wait for [the changelog updates](https://github.com/tinymce/tinymce/pull/6527) to go through before we merge this?